BWIndex.py

```diff
@@ -0,0 +1,239 @@
+from graphviz import Digraph
+import sys, argparse
+
+
+class BWIndex():
+	def __init__(self, genome=None, genomefile=None):

```

Comment:
I'm going to go ahead and just make lots of silly comments, warning you.

1) Python style guide recommends 4 space indents (and no tabs, boo). You can use a tool https://pypi.org/project/pep8/ to check adherence to PEP 8 (the style guide). I think there are even some that fix code to adhere to it.